README.md

```diff
@@ -2,16 +2,21 @@
 API
 ======
 
+A nodejs service that does the following:
```

Comment:
Do you think maybe it would be good to add a little background about the API's role before listing the specifics?

Reply:
What about this? Is there anything else that you'd add?

Reply:
Yup, I think that looks good, it's quite explanatory